tests/semantic_analyser.cpp

```diff
@@ -506,6 +507,23 @@ TEST(semantic_analyser, call_delete)
   test("kprobe:f { @x = 1; delete(@x) ? 0 : 1; }", 10);
 }
 
+TEST(semantic_analyser, call_delete_wildcard)
+{
+  test("kprobe:f { @x[0, 1] = 1; delete(@x[*, 1]); }", 0);
+  test("kprobe:f { @x[0, 1] = 1; $y = 1; delete(@x[* ,$y]); }", 0);
```

Comment:
Add a successful test case for:

* trailing wildcard (ie `delete(@x[1, *])`)
* multiple wildcards (ie `delete(@x[*, 1, *])`)